### DISCUSSION TITLE
How can I enable plugin for API

### DISCUSSION THREAD

**Discussion by Louvivien** [llm-workflow-engine/llm-workflow-engine#306] [Q&A]
Hello,

Thank you for this app
 I have read the readme but I can't figure how to enable GP4 with a specific plugin for the API?

**Comment by thehunmonkgroup** [comment]
As far as I know, ChatGPT Plugins and Browsing mode are only available natively via the web app, and is not available for access via the OpenAI API. Therefore, the API backend cannot support plugins or browsing mode natively. If I'm wrong, show me documentation to the contrary.

There is alpha support for plugins and browsing when using the browser backend, see https://github.com/mmabrouk/chatgpt-wrapper#using-chatgpt-plugins-alpha and https://github.com/mmabrouk/chatgpt-wrapper#using-chatgpt-with-browsing-alpha
